compiler/noirc_frontend/src/ast/expression.rs

```diff
@@ -46,6 +47,12 @@ pub enum ExpressionKind {
     Error,
 }
 
+impl ExpressionKind {
+    pub fn zero() -> Self {
```

Comment:
Let me know if it's okay to add these helper functions. We could also inline them.